react-form-controls/my-app/src/Uncontrolled.tsx

```diff
@@ -0,0 +1,25 @@
+import { FormEvent } from 'react';
+
+export function RegistrationFormUncontrolled() {
+  function handleSubmit(event: FormEvent) {
+    event.preventDefault();
+    const formData = new FormData(event.target as HTMLFormElement);
```

Comment:
To get the correct type, use `event.currentTarget`. Then you won't need the cast.